upgrade/upgrade-scripts/execute

```diff
@@ -28,6 +30,24 @@ function usage() {
 	exit 2
 }
 
+function generate_interface_to_mac_address_map() {
+	# we always create the map file anew to avoid using any stale versions of the same
+	if [[ -e "$INTERFACE_TO_MACADRESS_MAP_FILE_PATH" ]]; then
+		rm "$INTERFACE_TO_MACADRESS_MAP_FILE_PATH" ||
+			die "Could not delete the existing contents of the mac address map file"
+	fi
+	for i in $(find /sys/class/net/ -type l | awk -F "/" '{print $5}'); do
+		interface_name="$i"
+		if [ -e /sys/class/net/"$i"/address ] &&
+			grep -q -v -e "00:00:00:00:00:00" /sys/class/net/"$i"/address; then
+			mac_address=$(tr -d ":" </sys/class/net/"$i"/address)
```

Comment:
You might consider not doing that here given that the `:` is part of the MAC address. Instead, you could strip the `:` in the script that generates the right entries in `/etc/netplan`.

Reply:
Yes, I have moved the truncation to the update_netplan_to_mac_address script which is managed by the delphix-platform repository.